test/Microsoft.ML.Benchmarks/README.md

```diff
@@ -0,0 +1,36 @@
+# ML.NET Benchmarks
+
+This project contains performance benchmarks.
+
+## Run the Performance Tests
+
+**Pre-requisite:** On a clean repo, `build.cmd` at the root installs the right version of dotnet.exe and builds the solution. You need to build the solution in `Release` with native dependencies. 
+
+    build.cmd -release -buildNative
+
+**Pre-requisite:** To use dotnet cli from the `Tools\dotnetcli` directory remember to set `DOTNET_MULTILEVEL_LOOKUP` environment variable to `0`!
```

Comment:
This isn't essential, right? Any particular reason why I shouldn't use the machine one?

Reply:
I just double checked that and you are right, it's not mandatory.

Reply:
@danmosemsft fixed